app/src/main/java/com/dcrandroid/util/Utils.kt

```diff
@@ -165,10 +165,22 @@ object Utils {
         val multiWallet = WalletData.multiWallet!!
         val wallet = multiWallet.walletWithID(transaction.walletID)
 
-        val title = if (multiWallet.openedWalletsCount() > 1) {
-            context.getString(R.string.wallet_new_transaction, wallet.name)
-        } else {
-            context.getString(R.string.new_transaction)
+        val title = when (transaction.type) {
+            Dcrlibwallet.TxTypeVote -> context.getString(R.string.ticket_voted)
+            Dcrlibwallet.TxTypeRevocation -> context.getString(R.string.ticket_revoked)
```

Comment:
Add a variation to include the wallet's name if there's more than one wallet. Just like
https://github.com/planetdecred/dcrandroid/blob/09b0a0b3f15345389796cbb05428f234d070517d/app/src/main/java/com/dcrandroid/util/Utils.kt#L172-L176

Reply:
If there's more than one wallet, the wallet name is shown in the title of the notification.
do we still need to show the name before displaying the amount again?

Reply:
okay nvm, just realized to you mean for the title and not the amount